keyboards/dz60/keymaps/xtonhasvim/keymap.c

```diff
@@ -0,0 +1,94 @@
+#include "dz60.h"
+#include "xtonhasvim.h"
+
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
+
+enum layers {
+  _QWERTY,
+  _FUN,
+  _MOVE,
+  _MOUSE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[_QWERTY] = LAYOUT(
```

Comment:
This looks to be a standard 60% layout. You can use LAYOUT_60_ansi and you won't need to add KC_NO to your keymap.

Reply:
Yup, the board supports it: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/dz60/dz60.h#L94

Reply:
Except that I've got a split spacebar and `LAYOUT_60_ansi()` assumes not that.

Reply:
Ah, okay, then you should be fine, here.